docroot/themes/humsci/humsci_basic/templates/components/paragraph--hs-collection.html.twig

```diff
@@ -42,13 +42,17 @@
 {% if paragraph.field_raised_cards.value == true %}
    {% set raised_cards = true %}
 {% endif %}
+{% if paragraph.field_hs_collection_uh.value == true %}
+  {% set uniform_height = true %}
+{% endif %}
 {%
   set classes = [
     'paragraph',
     'paragraph--type--' ~ paragraph.bundle|clean_class,
     view_mode ? 'paragraph--view-mode--' ~ view_mode|clean_class,
     not paragraph.isPublished() ? 'paragraph--unpublished',
-    raised_cards ? 'hb-raised-cards hb-raised-cards--uniform-height'
+    raised_cards ? 'hb-raised-cards',
+    uniform_height ? 'hb-raised-cards--uniform-height'
```

Comment:
Please change this to

```
raised_cards and uniform_height ? 'hb-raised-cards--uniform-height'
```

For future-proofing, it's better to omit the uniform height class is the "raised cards" option is not set.

Reply:
Fixed